### DISCUSSION TITLE
Enhancement: Use 4o-mini instead of 3.5-turbo for chat labels

### DISCUSSION THREAD

**Discussion by nxfi777** [danny-avila/LibreChat#4233] [Q&A] [answered]
### What features would you like to see added?

Use 4o-mini instead of 3.5-turbo for chat labels to reduce user cost.

### More details

.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
See the relevant .env variables: https://www.librechat.ai/docs/configuration/dotenv#openai
